src/components/cookieToggle/index.js

```diff
@@ -0,0 +1,81 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import ToggleButton from '../toggleButton/index';
+import styles from './cookieToggle.module.scss';
+
+export default class CookieToggle extends Component {
+  static propTypes = {
+    header: PropTypes.string,
+    text: PropTypes.string,
+    cookies: PropTypes.arrayOf(PropTypes.shape({})),
+    isOn: PropTypes.bool,
+    disabled: PropTypes.bool,
+    handleToggleButton: PropTypes.func,
+    language: PropTypes.string,
+    id: PropTypes.string,
+  };
+
+  state = {
+    detailsOpen: false,
+  };
+
+  toggleDetails = () => {
+    this.setState(prevState => ({
+      detailsOpen: !prevState.detailsOpen,
+    }));
+  };
+
+  handleToggleButton = (isOn, id) => {
+    this.props.handleToggleButton(isOn, id);
+  };
+
+  render() {
+    const { header, isOn, disabled, text, cookies, id } = this.props;
+
+    return (
+      <div className={styles.cookieToggle}>
+        <div className={styles.header}>
+          <button className={styles.toggleDetails} onClick={this.toggleDetails}>
+            <FontAwesomeIcon
+              className={`${styles.chevron} ${
+                this.state.detailsOpen ? styles.open : ''
+              }`}
+              icon={['fas', 'chevron-right']}
+            />
+            <h6>{header}</h6>
+          </button>
+          <ToggleButton
+            id={id}
+            isOn={isOn}
+            disabled={disabled}
+            handleToggleButton={this.handleToggleButton}
+            language={this.props.language}
+          />
+        </div>
+        <p className="bodySmall">{text}</p>
+        <div
+          className={`${styles.tableWrapper} ${
+            this.state.detailsOpen ? styles.open : ''
+          }`}
```

Comment:
Jeg anbefaler på generelt basis å bruke `classNames` for lesbarhet. I dette eksempelet blir det:

```js
className={classNames(styles.tableWrapper, {
  [styles.open]: this.state.detailsOpen,
})}
```